platformio.ini

```diff
@@ -7,16 +7,20 @@
 ;
 ; Please visit documentation for the other options and examples
 ; https://docs.platformio.org/page/projectconf.html
+[platformio]
+default_envs = esp32doit
+
 [common_env_data]
 lib_deps = 
-    ;https://github.com/h2zero/NimBLE-Arduino.git#master //Nimble is tracked and including this will probably cause issues. 
```

Comment:
Should probably keep this here? I'm guessing we will eventually add it back in?

Reply:
Or is your plan to go completely cross platform dependent?

Reply:
Nope